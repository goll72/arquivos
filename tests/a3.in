4 ataque13.bin 2
1 idAttack 9999
1 targetIndustry "IT"

