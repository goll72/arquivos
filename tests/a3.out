174433.010000
